opencti-platform/opencti-front/src/schema/relay.schema.graphql

```diff
@@ -7786,8 +7786,8 @@ type Query {
   positions(first: Int, after: ID, orderBy: PositionsOrdering, orderMode: OrderingMode, filters: [PositionsFiltering], filterMode: FilterMode, search: String, toStix: Boolean): PositionConnection
   malware(id: String): Malware
   malwares(first: Int, after: ID, orderBy: MalwaresOrdering, orderMode: OrderingMode, filters: [MalwaresFiltering], filterMode: FilterMode, search: String, toStix: Boolean): MalwareConnection
-  threatActor(id: String): ThreatActor
-  threatActors(first: Int, after: ID, orderBy: ThreatActorsOrdering, orderMode: OrderingMode, filters: [ThreatActorsFiltering], filterMode: FilterMode, search: String, toStix: Boolean): ThreatActorConnection
+  threatActorGroup(id: String): ThreatActorGroup
+  threatActorsGroup(first: Int, after: ID, orderBy: ThreatActorsGroupOrdering, orderMode: OrderingMode, filters: [ThreatActorsGroupFiltering], filterMode: FilterMode, search: String, toStix: Boolean): ThreatActorGroupConnection
```

Comment:
I guess the python library would need to be updated ?

Reply:
Yes, it's updated on the "opencti/3166" branch in client-python depo